README.md

```diff
@@ -46,10 +46,10 @@ $ turnkey gen --name my-test-key
 }
 ```
 
-Sign a request:
+Make a request:
 
 ```sh
-$ turnkey approve-request --host api.turnkey.io --path /api/v1/sign --body '{"payload": "hello from TKHQ"}' --key=my-test-key
+$ turnkey request --host api.turnkey.io --path /api/v1/sign --body '{"payload": "hello from TKHQ"}' --key=my-test-key
```

Comment:
This example is missing the curl flag (the output includes the curl command, that's not the default for `turnkey request`)

Reply:
Oops!